maintainers/maintainer-list.nix

```diff
@@ -18250,6 +18250,12 @@
     githubId = 82817;
     name = "Robert Kreuzer";
   };
+  rksm = {
```

Comment:
The maintainers list change should be in a separate commit, with a message like `maintainers: add rksm`